src/networks.json

```diff
@@ -1539,6 +1539,20 @@
     "start": 7282345,
     "logo": "ipfs://QmRHB9TqMdVHY392vYiv8sTJ7VHShkq5FT6nS9fPuUNBf1"
   },
+  "7363": {
+    "key": "7363",
+    "name": "DynoChain",
```

Comment:
This block chain is not live yet?

Reply:
We are going live in 2 days. Etheruem chain list is taking forever to add our chain ids.